sdks/python/apache_beam/coders/typecoders.py

```diff
@@ -138,6 +137,10 @@ def get_coder(self, typehint):
         return coders.IterableCoder.from_type_hint(typehint, self)
       elif isinstance(typehint, typehints.ListConstraint):
         return coders.ListCoder.from_type_hint(typehint, self)
+      elif (isinstance(typehint, typehints.UnionConstraint) and
+            typehint.contains_type(type(None) and
+                                   len(list(typehint._inner_types())) == 2)):
+        return coders.NullableCoder.from_type_hint(typehint, self)
```

Comment:
Did you mean to use the inner type in this call ?

Reply:
No, I thought it made more sense to use the full typehint, to try and match the existing coder.from_type_hint pattern